test/Uri.js

```diff
@@ -48,8 +48,39 @@ describe('Uri', function() {
 		assert.strictEqual('', uri.getOrigin());
 	});
 
+	it('should set protocol http: on localhost uri', function() {
+		var uri = new Uri('localhost:8080');
+		assert.strictEqual('http:', uri.getProtocol());
+		assert.strictEqual('8080', uri.getPort());
+	});
+
+	it('should support useful types of uri schemes like "tel"', function() {
+		var uri = new Uri('tel:hostname');
+		assert.strictEqual('tel:', uri.getProtocol());
+	});
+
+	it('should support uri schemes that include hyphens', function() {
+		var uri = new Uri('ms-excel:hostname');
+		assert.strictEqual('ms-excel:', uri.getProtocol());
+	});
+
+	it('should support uri schemes that include numerical values', function() {
+		var uri = new Uri('pkcs11:hostname');
+		assert.strictEqual('pkcs11:', uri.getProtocol());
+	});
+
+	it('should support uri schemes that include periods', function() {
+		var uri = new Uri('iris.beep:hostname');
+		assert.strictEqual('iris.beep:', uri.getProtocol());
+	});
+
+	it('should support uri schemes that include pluses', function() {
+		var uri = new Uri('some+scheme:hostname');
+		assert.strictEqual('some+scheme:', uri.getProtocol());
+	});
+
 	it('should support set port on uri', function() {
-		var uri = new Uri('hostname:8080');
+		var uri = new Uri('http://hostname:8080');
```

Comment:
Hey guys, sorry for the delay.

@jbalsas, Senna will not navigate to a `url` that has not the same origin of the website, once `isLinkSameOrigin_` validates it. But `canNavigate()` throws an error to a link tag that has this `href` value: `tel:123-123-1234`, once `metal-uri` don't recognize `tel:` as an protocol and adds `http:`. The string will look like this: `http:tel:123-123-1234`.

I was talking to @brunobasto and we can prevent Senna to throws the error with a `try/catch`. This way it's not necessary to change `metal-uri` behaviour, once the component was developt to deal only with `http` protocol.